src/molecules/Select/README.md

```diff
@@ -26,23 +35,15 @@ Table below contains all types of props available in the Select component
 
 | Name                  | Type       | Default         | Description                      |
 | :------------         | :-----     | :-------------- | :------------------------------- |
-| **children**          | `Function` |                 | Render the `Option`s for the select
-| **placeholder**       | `string`   |                 | The placeholder/label to display in the select field
+| **options**           | `Function` |                 | An array of options to render in the select
+| **placeholder**       | `string`   |                 | The placeholder/label to display before a value has been selected
```

Comment:
Just a thought on this - does a standard select show the top option in the list as the placeholder or even like an empty '-' to deselect all options - for instance if you want to remove the fact you've selected something

Reply:
Hmm not yet, it's a good point tho! Probably worth creating a ticket for that in the backlog and implement it when needed?